src/components/charts/galaxySelector/blinker/BlinkerControls.jsx

```diff
@@ -20,19 +21,19 @@ const BlinkerControls = ({
       <Button
         className={control}
         icon
-        iconEl={<Rewind />}
+        iconEl={<ButtonIcon srText="Rewind" Icon={Rewind} />}
         onClick={handlePrevious}
       />
       <Button
         className={control}
         icon
-        iconEl={<StartStopTag />}
+        iconEl={<ButtonIcon srText="Start/Stop" Icon={StartStopTag} />}
         onClick={handleStartStop}
       />
       <Button
         className={control}
         icon
-        iconEl={<FastForward />}
+        iconEl={<ButtonIcon srText="Fast Forward" Icon={FastForward} />}
```

Comment:
This seems to be working well.  THe other spot that could use this treatment is the mini NavDrawer menu items.  Those all just read out as "Toggle button", as well as the button that opens and closes the full sized NavDrawer.  The latter seems to have a span inside it that should read out in a screen reader correctly, but doesn't for some reason. The case of the former is more critical than the latter as the fullsized NavDrawer is not in fact accessible at all, but that's my bad, an oversight I'll have to deal with at a later date.  But for now, if possible, I think we'd like to see those icon only NavDrawer items get screen reader only text if at all possible within the confines of our react-md world.